jahia-test-module/src/helpers/menuEntryCss.ts

```diff
@@ -1,4 +1,5 @@
-export const menuEntryCss = (menu, isFirst, isLast) => {
+import type { MenuEntry } from "@jahia/javascript-modules-library";
+export const menuEntryCss = (menu: MenuEntry, isFirst: boolean, isLast: boolean) => {
```

Comment:
Damn that's exactly the issue https://www.npmjs.com/package/clsx solves

Keep it as is for now